Source/WebCore/Modules/modern-media-controls/controls/media-controls.js

```diff
@@ -183,9 +183,9 @@ class MediaControls extends LayoutNode
             return false;
 
         const tappedElement = shadowRoot.elementFromPoint(point.x, point.y);
-
-        if (includeContainer && this.element === tappedElement)
-            return true;
+        
+        if (tappedElement.classList.contains("backdrop"))
```

Comment:
Can we set `pointer-events: none` on the `backdrop` element instead?

Reply:
I'll try it!